editor/plugins/script_editor_plugin.cpp

```diff
@@ -1380,33 +1380,57 @@ void ScriptEditor::_update_script_names() {
 	_update_script_colors();
 }
 
-void ScriptEditor::edit(const Ref<Script> &p_script, bool p_grab_focus) {
+bool ScriptEditor::edit(const Ref<Script> &p_script, int p_line, int p_col, bool p_grab_focus) {
 
 	if (p_script.is_null())
-		return;
+		return false;
 
 	// refuse to open built-in if scene is not loaded
 
 	// see if already has it
 
 	bool open_dominant = EditorSettings::get_singleton()->get("text_editor/files/open_dominant_script_on_scene_change");
 
+	Error err = p_script->get_language()->open_in_external_editor(p_script, p_line >= 0 ? p_line : 0, p_col);
```

Comment:
Same here. First char in a file is 1:1, so 0 as line *or* colum already would be invalid.